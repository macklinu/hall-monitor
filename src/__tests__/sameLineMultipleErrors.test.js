import hallMonitor from '..'

const rule = context => ({
  VariableDeclaration(node) {
    if (node.kind === 'var') {
      context.report({
        node,
        message: 'No vars allowed',
      })
    }
  },
})

hallMonitor('same-line-multiple-errors', rule, {
  valid: ['const a = 1'],
  invalid: ['var a; var b; var c;', 'var a; var b; var c;\nvar d;'],
})
