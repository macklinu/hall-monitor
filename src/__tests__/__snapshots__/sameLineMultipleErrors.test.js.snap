// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`same-line-multiple-errors invalid var a; var b; var c;
var d; 1`] = `
"var a; var b; var c;
               ^^^^^^ [No vars allowed]
 ^^^^^^ [No vars allowed]
        ^^^^^^ [No vars allowed]
 ^^^^^^ [No vars allowed]
var d;"
`;

exports[`same-line-multiple-errors invalid var a; var b; var c; 1`] = `
"var a; var b; var c;
               ^^^^^^ [No vars allowed]
        ^^^^^^ [No vars allowed]
 ^^^^^^ [No vars allowed]"
`;

exports[`same-line-multiple-errors valid const a = 1 1`] = `"const a = 1"`;
